creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -474,8 +478,11 @@ const _selectRandomReplicaSetNodes = async (
 
   const newReplicaNodesSet = new Set<string>()
 
+  const spInfoMap = await getMapOfCNodeEndpointToSpId(logger)
   const viablePotentialReplicas = healthyNodes.filter(
-    (node) => !healthyReplicaSet.has(node)
+    (node) =>
+      !healthyReplicaSet.has(node) &&
+      !RECONFIG_SP_IDS_BLACKLIST_SET.has(spInfoMap.get(node))
```

Comment:
i think this is a fine way to do it but [autoSelectCreatorNodes](https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/api/ServiceProvider.ts#L25) would do this for you if you add the blacklist property [here](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts#L130)

this way the selection would return the healthy, not blacklisted nodes and we wouldn't need to add this filter everywhere

Reply:
done